### ISSUE TITLE
SyntaxError: Unexpected token )

### ISSUE THREAD

**Issue by sqrthree** [geowarin/friendly-errors-webpack-plugin#35]
when I installed version 1.6.0, I got this follow error:

```
vue-demo/node_modules/friendly-errors-webpack-plugin/src/formatters/moduleNotFound.js:39
  );
  ^
SyntaxError: Unexpected token )
    at Object.exports.runInThisContext (vm.js:73:16)
    at Module._compile (module.js:543:28)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/Users/sqrtthree/Downloads/vue-demo/node_modules/friendly-errors-webpack-plugin/src/friendly-errors-plugin.js:21:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
```
I think this is because that there is a extra comma behind the last argument in line [38](https://github.com/geowarin/friendly-errors-webpack-plugin/blob/master/src/formatters/moduleNotFound.js#L38).

**Comment by christiankossler** [comment]
Got this too. This error also affects rows 39, 49, 83.

**Comment by Hendriksie** [comment]
Got the same error after updating,

The reason this is happening is the last comma `,` in the `concat ` function.

As @christiankossler said, it affects rows 39, 49 and 83

**Comment by disitec** [comment]
Same here. I had to comment the import module in vue-cli.

**Comment by laurentpayot** [comment]
+1

**Comment by pratikborsadiya** [comment]
:+1:

**Comment by xiguaxigua** [comment]
+1

**Comment by alaj** [comment]
+1

**Comment by simonmurdock** [comment]
+1 currently breaking the latest vue JS

**Comment by alaj** [comment]
I use the previous version 1.50 and ok for now. You can try  'npm install friendly-errors-webpack-plugin@1.50 --save'

**Comment by simonmurdock** [comment]
I think you mean
`` npm install friendly-errors-webpack-plugin@1.50 --save``
@alaj

**Comment by alaj** [comment]
@simonmurdock 
Yes. That's what I did. ^_^

**Comment by geowarin** [comment]
Fixed in 1.6.1
